kernel/driver_8254.asm

```diff
@@ -0,0 +1,47 @@
+    ; 8254 Programmable Timer drivers.
+    .equ    TIMER_0, 0x10
+    .equ    TIMER_CTRL, 0x13
+
+    ; At 3.686400 MHz each cycle is ~0.27us.
+    ;
+    ; Using a count of 61,440 results in a period of:
+    ;     16.6ms
+    ;
+    ; We context-switch every 6 counts, resulting in
+    ; a period of:
+    ;     99.999999959ms
+    ;
+    ; Ideal period is 100ms therefore error is:
+    ;     0.041ns
+    .equ    TIMER_COUNT, 61440
+
+    ; void timer_init(void)
```

Comment:
For the sake of completeness, this initialization function should initialize all three timers (even though timers 1 and 2 are unused)

Reply:
Writing a command word in mode 0 to a timer without following up with count bytes will put it in an "inactive" state (i.e. not triggering interrupts)